### PR TITLE
Start building with CircleCI

### PR DESCRIPTION
**Pull request by krisalyssa**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Elixir CircleCI 2.0 configuration file
```
Comment (jeffutter) [inline .circleci/config.yml:1]: We have had _really_ bad experiences with Circle CI 2.0 (compared to 1.0) There is a lot of upheaval in their community about it missing features and being unstable. 

That said, this config is pretty simple. If it works, it works. Just keep that in mind if we start having trouble and don't be afraid to switch to Circle CI 1.0 or another platform.

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      # documented at https://circleci.com/docs/2.0/circleci-images/
+      # - image: circleci/postgres:9.4
+
+    working_directory: ~/repo
+    steps:
+      - checkout
+      - run: mix local.hex --force
+      - run: mix local.rebar --force
+      - run: mix deps.get
+      - run: mix test
```
Comment (jeffutter) [inline .circleci/config.yml:22]: Can you add: 

```
- run: mix format --check-formatted --dry-run
- run: mix credo
```

**Review by jeffutter — CHANGES_REQUESTED** [review]
Just a couple more tests to add.

File: .circleci/config.yml (hunk 2 above)
Comment (krisalyssa) [reply .circleci/config.yml:22]: Done.

**Review by jeffutter — APPROVED** [review]

**Pull request merged**